### ISSUE TITLE
Charge is not included when rendering in concise formula

### ISSUE THREAD

**Issue by tomyan112** [Chem4Word/Version3-2#38]
**Describe the bug**

![image](https://user-images.githubusercontent.com/9064351/157408231-b6f85545-dda0-4da8-a496-016cd844272c.png)

**Expected behavior**
Render as [PtCl₄]²⁻ instead of PtCl₄

Edit: Put back xml in markdown
**CML Data**
```xml
<cml:cml xmlns:conventions="http://www.xml-cml.org/convention/" xmlns:cml="http://www.xml-cml.org/schema" xmlns:cmlDict="http://www.xml-cml.org/dictionary/cml/" xmlns:nameDict="http://www.xml-cml.org/dictionary/cml/name/" xmlns:c4w="http://www.chem4word.com/cml" conventions="convention:molecular">
  <c4w:customXmlPartGuid>a7c4511adb8842e5b6f8739db1caf8d3</c4w:customXmlPartGuid>
  <cml:molecule id="m1" formalCharge="-2">
    <cml:formula id="m1.f0" concise="Pt 1 Cl 4" />
    <cml:formula id="m1.f1" convention="chem4word:ResolvedFormula" inline="Cl4Pt" concise="Pt 1 Cl 4" />
    <cml:formula id="m1.f2" convention="chem4word:ResolvedSmiles" inline="Cl[Pt](Cl)(Cl)Cl" concise="Pt 1 Cl 4" />
    <cml:name id="m1.n1" dictRef="chem4word:CalculatedInchi">1S/4ClH.Pt/h4*1H;/q;;;;+4/p-4</cml:name>
    <cml:name id="m1.n2" dictRef="chem4word:CalculatedInchikey">FBEIPJNQGITEBL-UHFFFAOYSA-J</cml:name>
    <cml:name id="m1.n3" dictRef="chem4word:ResolvedIupacname">tetrachloroplatinum</cml:name>
    <cml:atomArray>
      <cml:atom id="a1" elementType="Pt" x2="634.7596" y2="610.0" />
      <cml:atom id="a2" elementType="Cl" x2="613.5464" y2="588.7868" />
      <cml:atom id="a3" elementType="Cl" x2="655.9728" y2="588.7868" />
      <cml:atom id="a4" elementType="Cl" x2="608.7788" y2="625.0" />
      <cml:atom id="a5" elementType="Cl" x2="660.7404" y2="625.0" />
    </cml:atomArray>
    <cml:bondArray>
      <cml:bond id="b1" atomRefs2="a1 a2" order="S">
        <cml:bondStereo atomRefs2="a1 a2">H</cml:bondStereo>
      </cml:bond>
      <cml:bond id="b2" atomRefs2="a1 a3" order="S">
        <cml:bondStereo atomRefs2="a1 a3">H</cml:bondStereo>
      </cml:bond>
      <cml:bond id="b3" atomRefs2="a4 a1" order="S">
        <cml:bondStereo atomRefs2="a4 a1">W</cml:bondStereo>
      </cml:bond>
      <cml:bond id="b4" atomRefs2="a1 a5" order="S">
        <cml:bondStereo atomRefs2="a1 a5">W</cml:bondStereo>
      </cml:bond>
    </cml:bondArray>
  </cml:molecule>
</cml:cml>
```

**Comment by MikeWilliams-UK** [comment]
Thanks @tomyan112 I will add that to our internal backlog.

**Comment by MikeWilliams-UK** [comment]
Included in V3.2.3 Beta 3
